.github/workflows/dev-test-run.yml

```
@@ -0,0 +1,66 @@
+
+    strategy:
+      matrix:
+        node-version: "20"
+
+    defaults:
+      run:
+        working-directory: web/e2e/playwright-tests
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)